critiquebrainz/frontend/external/musicbrainz.py

```diff
@@ -73,6 +73,27 @@ def browse_release_groups(artist_id=None, release_types=None, limit=None, offset
     return release_groups
 
 
+def browse_releases(artist_id=None, release_types=None, limit=None, offset=None, release_group=None, includes=None):
+    """Get all the releases by a certain artist.
```

Comment:
I'm talking about the line I commented on. It doesn't seem to be describing what the function does correctly. Please check that.